### ISSUE TITLE
Panel title and legend are blurry

### ISSUE THREAD

**Issue by Pierretherenard** [bokeh/bokeh#5474]
Hi ,
All widget, tabs, labels are crispy clear but the titles of any figure are blurry.
Here is the code I use, nothing fancy. Does that mean There are some font options that I need to use to match the clarity of the other elements? or do I need new glasses?

Thank you,
Pierre
![screen shot 2016-11-18 at 9 16 49 am](https://cloud.githubusercontent.com/assets/23002446/20439302/c7ec86f6-ad6f-11e6-9214-daa1d9472465.png)

`pl = figure(plot_width=900, plot_height=650, title="GJT frequency",
                       tools="pan,wheel_zoom,box_zoom,crosshair,reset,hover,previewsave", x_axis_type="datetime")`
![screen shot 2016-11-18 at 9 10 27 am](https://cloud.githubusercontent.com/assets/23002446/20439088/e669e49e-ad6e-11e6-8b9d-c778e517ec2b.png)

![screen shot 2016-11-16 at 6 16 03 pm](https://cloud.githubusercontent.com/assets/23002446/20439038/aa3c1c44-ad6e-11e6-96f6-01ea21b03495.png)
The issue tracker isn't the place for general support. For questions and
technical assistance, come ask the [Bokeh mailing list](https://groups.google.com/a/continuum.io/forum/#!forum/bokeh) or join the chat on [Gitter](https://gitter.im/bokeh/bokeh).

For defects or deficiencies, please provide:
* Description of expected behavior and the observed behavior
* Self-contained example code that generates the issue (if possible)
* Stack traceback and/or browser developer tools console output
* Software version information output by `bokeh info` command

For desired features, please provide:
* Description of the desired feature

**Comment by Pierretherenard** [comment]
oups, in the title Not Panel, Figure

**Comment by bryevdv** [comment]
@Pierretherenard I have not encountered this on the platforms I regularly use, so this is likely something that is platform specific. Please provide details about hardware, OS and version, browser and version, etc.

**Comment by Pierretherenard** [comment]
Mac Pro late 2011
intel HD Graphic 3000
safari 10.0.1
OS X Yosemite 10.10.5
bokeh 0.12.3

**Comment by bryevdv** [comment]
@Pierretherenard I'm not sure where I'd be able to procure a 2011 mac pro to test with. By way of gathering more information, do you have the ability to try your example on other systems and configurations and report whether or not it is reproducible there?

**Comment by Pierretherenard** [comment]
I did try 
`show(tabs,browser=Firefox)`, 
but it did continue to open in Safari anyway. Is this the correct line to open on another browser? Then l opened the html file with Firefox (50.0) with my little hand and the blurry is **NOT** there!

**Comment by Pierretherenard** [comment]
I had other problems as well with safari, like datable suddenly being blank or sticky mouse trace and they all disappeared with Firefox, way more stable with Firefox on my machine.

**Comment by Pierretherenard** [comment]
I am not able to test on another platform right now. The display has no problem with Firefox. It is strange because it is only happening with Figure Title and Legend, all others are fine.
Thank you,
Pierre

**Comment by Pierretherenard** [comment]
I am reopening this one because I think i found something that narrow where this problem might come from. be back later

**Comment by Pierretherenard** [comment]
I tried to recreate the display problem. Before it did happen right away as soon as I moved the cursor on the figure but it does not happen anymore, not sure why, but: If I Zoom on **safari**  this is what happen:
![screen shot 2017-01-02 at 8 35 36 pm](https://cloud.githubusercontent.com/assets/23002446/21600095/bf4b8b66-d12b-11e6-89ce-b3a35d32a12d.png)
This never happen on **firefox** even if I Zoom.
Yes this is still on the same hardware configuration.
